source/publishing_data/05_processing_data/processors/correct_geo_shape.rst

```diff
@@ -0,0 +1,24 @@
+Correct geo shape processor
+===========================
+
+.. admonition:: Important
+  :class: important
+
+  This processor is not available by default. Please contact Opendatasoft support team if you want this processor to be activated in your domain.
+
+This processor fixes geo shapes with invalid ring-intersection and geo shapes mixing 2D and 3D points.
```

Comment:
We may want to temper that assertion. The processor fixes the geo shapes but sometimes it's not valid anyway.
It's rare but it happens but maybe it's not relevant to write that in the documentation 🤔

Reply:
Depending on whether we're confident in the processor or not, we could slightly change the docs to:
"This processor **attempts to fix** geo shapes with invalid ring-intersection and geo shapes mixing 2D and 3D points."

Reply:
Let's keep it like that.